presto-main/src/main/java/com/facebook/presto/operator/WindowOperator.java

```diff
@@ -387,6 +403,9 @@ private Page extractOutput()
     {
         // INVARIANT: pagesIndex contains the full grouped & sorted data for one or more partitions
 
+        // Index is already prepared
+        windowInfo.addIndex(pagesIndex);
```

Comment:
how big can it grow? What if pages are really small. My concern is that there might be millions of `IndexInfos` in `SingleDriverWindowInfoBuilder`, which is not acceptable